packages/core/src/utils/regex.ts

```diff
@@ -3,4 +3,5 @@ export const phoneRegEx = /^[1-9]\d{10}$/;
 export const usernameRegEx = /^.{3,}$/;
 export const nameRegEx = /^.{3,}$/;
 export const passwordRegEx = /^.{6,}$/;
+export const redirectUriRegEx = /^https?:\/\//;
```

Comment:
is it has to be a HTTP redirect address? could it be a native schema?

Reply:
This RegExp was initially for Alipay only as URLs starting with HTTP(s) are required as the attached screenshot shows. As the route which checks redirectUri format might also be used by mobile ends, we might move this check back to the Alipay web field? @wangsijie 
<img width="1800" alt="image" src="https://user-images.githubusercontent.com/15182327/163095755-d9a44ec6-6be3-44c7-a56b-6a51ae0bde40.png">